scalafix-sbt/src/main/scala/sbt/scalafixsbt/JLineAccess.scala

```diff
@@ -0,0 +1,6 @@
+package sbt.scalafixsbt
+
+/** Helper class to access sbt's JLine instance */
+object JLineAccess {
+  def terminalWidth: Int = sbt.internal.util.JLine.usingTerminal(_.getWidth)
+}
```

Comment:
This `terminalWidth` depends on sbt's internal JLine ... and is necessary for sbt 0.13.* and sbt 1.* compatibility.

We should use [sbt.CommandLineUIService#terminalWidth](https://github.com/sbt/sbt/commit/f96c8b966874da82e5b6ea6f095bf4826bfe2c4e) instead, but it seems to be implemented only in sbt 1.* .